scripts/zkai_ezkl_statement_envelope_benchmark.py

```
@@ -0,0 +1,502 @@
+            }
+            for adapter in adapters
+        },
+    }
+
+
+def _git_commit() -> str:
+    import subprocess
+
+    try:
+        return subprocess.check_output(
+            ["git", "rev-parse", "HEAD"],
```

Ruff (0.15.12)

[error] Starting a process with a partial executable path

(S607)